### ISSUE TITLE
Avoid alternative text that is the same as adjacent text

### ISSUE THREAD

**Issue by gilleygoat** [web-infrastructure/doc_architecture#36]
An `<img>` element inside a link must not specify alternative text that duplicates the content of a text link inside or beside it.

For example, this code should be avoided:
```css
<a href="/apple">
    <img src="apple.png" alt="Apple">
    Apple
</a>
```
Accessible technologies will tend to see this as "Apple Apple". The correct behaviour is to specify either a null (empty) `alt` attribute:
```css
<a href="/apple">
    <img src="apple.png" alt="">
    Apple
</a>
```
or to hide the image via aria-hidden:
```css
<a href="/apple">
    <img src="apple.png" aria-hidden="true">
    Apple
</a>
```
Learn more

Read about [Technique H2](https://www.w3.org/TR/WCAG20-TECHS/H2) on how to combine an adjacent image and texts links.

See the [WAI Tutorial on decorative images](https://www.w3.org/WAI/tutorials/images/decorative/).
